api/client/nic.cc

```diff
@@ -358,19 +358,23 @@ NicAddOptions::NicAddOptions() {
     bypass_filtering = "false";
 }
 
+static inline bool IsArg(int count, int argc, char **argv, char const *option) {
```

Comment:
really bad name, this function doesn't not check if a string is an arg, it check if there is an arg, and if this arg correspond to the option.
maybe CheckOption